Tests/FunctionalTests/AEPEdgeFunctionalTests.swift

```diff
@@ -34,16 +34,13 @@ class AEPEdgeFunctionalTests: FunctionalTestBase {
         continueAfterFailure = false
         FileManager.default.clearCache()
 
-        // hub shared state update for 2 extension versions (InstrumentedExtension (registered in FunctionalTestBase), Identity, Edge), Identity and Config shared state updates
-        setExpectationEvent(type: FunctionalTestConst.EventType.HUB, source: FunctionalTestConst.EventSource.SHARED_STATE, expectedCount: 4)
+        // hub shared state update for 2 extension versions (InstrumentedExtension (registered in FunctionalTestBase), IdentityEdge, Edge), IdentityEdge standard, IdentityEdge XDM and Config shared state updates
```

Comment:
cc @lind I think we will not need the standard shared state for IdentityEdge anymore

Reply:
Yes, I'm working on some changes which removes the regular shared state from IdentityEdge. I'll have a PR later this week.

Reply:
IdentityEdge dev no longer sets a standard shared state. Can you update the Pods and change the test expectations?